doc/dev/release-process.rst

```diff
@@ -223,20 +223,56 @@ mv the directories and the tarballs from the prerelease home
 5. Build Containers
 ===================
 
-Architecture-specific containers are built during the ceph build and
-pushed to quay.ceph.io/ceph/prerelease-{amd64,arm64}, containing the
-packages built in that ceph build.  The prerelease 'fat' container,
-or manifest-list container, that refers to both arch-specific containers,
-is built by hand using the command "make-manifest-list.py" in
-ceph.git:src/container/make-manifest-list.py.  Note that you must
+Unlike CI builds, which have access to packages in the correct form for
+the container, release builds do not, because the build does not 
+sign the packages.  Thus, release builds do not build the containers.
+This must be done after :ref:`Signing and Publishing the Build`.
+
+Architecture-specific containers are built first, and pushed to
+quay.ceph.io/ceph/prerelease-{amd64,arm64}.  Note: this must be done on
+both architectures.
+
+* Use a host with a relatively-recent version of podman and skopeo
+available.  CentOS/RHEL/Fedora usually have later versions than Ubuntu,
+but Ubuntu 22.04 or later are probably ok.
+
+* Copy and run this shell wrapper for building a container (in
+container/ is assumed below, to invoke ./build.sh), replacing the
+values in <> as appropriate:
+
+     .. prompt:: bash
```

Comment:
I suspect that line 243 must be followed with a blank line.

Reply:
thanks, I'm not sure why I didn't even try a local build

Reply:
I can clean it up and merge it if you give me the go-ahead.

Reply:
Fixed up a bunch of nonsense.  I still have an issue with one of the numbered lists restarting in BUILD CONTAINERS, if you can fix that, that'd be awesome.  I thought a lot of the problems were because ``#.`` needed to have all its item on one line, but that doesn't seem to have done it for the last one.

Any English fixups welcomed, too.

Reply:
I'll have a crack at it. Check back in half an hour.

Reply:
@dmick, I have made two force-pushes to this branch. One fixes the RST governing the ordered list in the section called "Build Containers". The other adds markup to "Build Containers" where I judged the addition would make the text easier to read.

The commits are:

1124dcd8eefec10802f0f1f3d0c67debc6f71f58 - list repair
fcbf6de0f1c6772a9cbe3009426d660d1357ec83 - markup and grammar

If you are happy for me to squash these into your commit, I will do so.

I await your yea or nay.